client/app/components/form/FormBase.tsx

```diff
@@ -30,18 +31,28 @@ const validator = customizeValidator({ customFormats });
 interface FormPropsWithTheme<T> extends Omit<FormProps<T>, "validator"> {
   theme?: ThemeProps;
   validator?: any;
+  setErrorReset?: (error: undefined) => void;
 }
 
 const FormBase: React.FC<FormPropsWithTheme<any>> = (props) => {
-  const { theme, formData, omitExtraData } = props;
+  const { theme, formData, omitExtraData, setErrorReset } = props;
   const Form = useMemo(() => withTheme(theme ?? defaultTheme), [theme]);
+  const [formState, setFormState] = useState(formData ?? {});
```

Comment:
Looks like this needs a rebase, all of this is from a PR that recently was merged in.